usng4j-api/src/main/java/org/codice/usng4j/UsngCoordinate.java

```diff
@@ -59,7 +59,7 @@
   public final String USNG_COORDINATE_PART_REGEX_STRING = "(\\W\\d{0,5})?(\\W\\d{0,5})?";
 
   /** RegEx expressions for MGRS Northing and Easting parsing */
-  public final String MGRS_COORDINATE_PART_REGEX_STRING = "(\\d{0,5})\\W*(\\d{0,5})\\W*";
+  public final String MGRS_COORDINATE_PART_REGEX_STRING = "(\\d{0,12})\\W*";
```

Comment:
What non-word characters are we hoping to match here?

Reply:
There shouldn't be any other characters to match after the numerical location. I changed the regex to only look for up to 10 digits.